src/components/PaymentInfoLine.vue

```diff
@@ -1,41 +1,87 @@
 <template>
-    <div class="info-line">
-        <Amount :amount="amount + fee" :decimals="decimals" />
+    <div class="info-line" :class="{ 'inverse-theme': theme === constructor.Themes.INVERSE }">
+        <div class="amounts">
+            <Amount
+                :currency="cryptoAmount.currency"
+                :amount="cryptoAmount.amount"
+                :totalDecimals="cryptoAmount.digits"
+                :minDecimals="0"
+                :maxDecimals="Math.min(4, cryptoAmount.digits)"
```

Comment:
The max decimals should propably be dynamic based on the crypto currency they represent. E.g. for ETH, 4 makes sense, but for BTC it should propably be 6, and for NIM it can be 2 or even 0.

Could also be based on the price, to make it totally dynamic. Ref:
https://github.com/nimiq/woocommerce-gateway-nimiq/blob/e8985b1dc18e46f8266a63dc6ecac38a2ae3b6c2/includes/crypto_manager.php#L52-L55

Reply:
I don't think this is necessary.